src/components/how-to-card/HowToCard.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import './HowToCard.css';
+
+export default function HowToCard({ step, title, subtitle, image, alt }) {
```

Comment:
Looks good! For the Requests component I didn't create a separate file for the cards though (cause it's bascially just a heading and a container with 3 cards). Do you think I should do that too?

Reply:
Nah I think it's ok. If you want to go for it but it's not a breaking issue!